docs/en/query_language/table_functions/index.md

```diff
@@ -1,8 +1,30 @@
 # Table functions
 
-Table functions can be specified in the FROM clause instead of the database and table names.
-Table functions can only be used if 'readonly' is not set.
-Table functions aren't related to other functions.
+Table function is a method of creating a table.
```

Comment:
I'd use some different verb instead of "create" to avoid implying that it's permanent.